src/MusicalScore/Graphical/VexFlow/VexFlowMeasure.ts

```diff
@@ -1394,19 +1404,46 @@ export class VexFlowMeasure extends GraphicalMeasure {
             return;
         }
         const vexFlowVoiceEntry: VexFlowVoiceEntry = voiceEntry as VexFlowVoiceEntry;
-        const note: GraphicalNote = voiceEntry.notes[0]; // only display for top note
-        const stringInstruction: TechnicalInstruction = note.sourceNote.StringInstruction;
-        if (stringInstruction) {
-            const stringNumber: number = Number.parseInt(stringInstruction.value, 10);
-            const vfStringNumber: Vex.Flow.StringNumber = new Vex.Flow.StringNumber(stringNumber);
-            const offsetY: number = -this.rules.StringNumberOffsetY;
-            // if (note.sourceNote.halfTone < 50) { // place string number a little higher for notes with ledger lines below staff
-            //     // TODO also check for treble clef (adjust for viola, cello, etc)
-            //     offsetY += 10;
-            // }
-            vfStringNumber.setOffsetY(offsetY);
-            vexFlowVoiceEntry.vfStaveNote.addModifier((0 as any), (vfStringNumber as any)); // see addModifier() above
-        }
+        voiceEntry.notes.forEach((note, stringIndex) => {
+            const stringInstruction: TechnicalInstruction = note.sourceNote.StringInstruction;
+            if (stringInstruction) {
+                let stringNumber: string = stringInstruction.value;
+                switch (stringNumber) {
+                    default:
+                    case "1":
+                        stringNumber = "I";
+                        break;
+                    case "2":
+                        stringNumber = "II";
+                        break;
+                    case "3":
+                        stringNumber = "III";
+                        break;
+                    case "4":
+                        stringNumber = "IV";
```

Comment:
by the way, we should probably support string numbers up to at least 6 / "VI",
e.g. for the guitar.
i'll just add that after merging i think.